src/_guides/language/effective-dart/usage_migrated.md

```diff
@@ -711,6 +711,24 @@ void error([String? message = null]) {
 
 The following best practices describe how to best use variables in Dart.
 
+### DO follow a consistent rule for `var` and `final` on local variables.
+
+Most local variables should not have type annotations and should be declared
```

Comment:
We generally use "n't" instead of "not": https://developers.google.com/style/contractions